InnerEye/ML/configs/histo_configs/classification/DeepSMILEPanda.py

```diff
@@ -76,13 +77,15 @@ def cache_dir(self) -> Path:
 
     def setup(self) -> None:
         if self.encoder_type == InnerEyeSSLEncoder.__name__:
+            from InnerEye.ML.configs.histo_configs.run_ids import innereye_ssl_checkpoint_binary
             self.downloader = CheckpointDownloader(
-                azure_config_json_path=get_workspace(),
-                run_recovery_id=innereye_ssl_checkpoint,
-                checkpoint_filename="last.ckpt",
+                aml_workspace=get_workspace(),
+                run_id=innereye_ssl_checkpoint_binary,  # innereye_ssl_checkpoint
+                checkpoint_filename="best_checkpoint.ckpt",  # "last.ckpt",
                 download_dir="outputs/",
+                remote_checkpoint_dir=Path("outputs/checkpoints")
             )
-            os.chdir(fixed_paths.repository_root_directory())
+            os.chdir(fixed_paths.repository_root_directory().parent)
```

Comment:
That's the same snippet as in DeepSmileCrck, re-use?

Reply:
it's not exactly the same - the parameters we pass to checkpoint downloader are different and the run_id variable we import is different. We could make all these parameters of setup but I am not sure this is really needed as part of this PR.  In principle I think the setup is heavily experiment dependent and it seems fair to have  base class that explicitly requires he user to define their own setup method if a checkpoint is needed. Especially given these checkpoints are not made publicly available, that implies the user will need to define their own container anyway. What you think?